qemu/tests/qemu_guest_agent.py

```diff
@@ -338,7 +335,7 @@ def gagent_check_powerdown(self, test, params, env):
         """
         self.__gagent_check_shutdown(self.gagent.SHUTDOWN_MODE_POWERDOWN)
         if not utils_misc.wait_for(self.vm.is_dead, self.vm.REBOOT_TIMEOUT):
-            raise error.TestFail("Could not shutdown VM via guest agent'")
+            test.fail("Could not shutdown VM via guest agent'")
```

Comment:
`self.test`

also applies for the ones below.

Reply:
Thanks, updated